### ISSUE TITLE
Suggestion, if possible read the table of disk partitions in the disk manager

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#7047]
Hello everyone.
I do not know if my suggestion is possible to implement.
When in disk manager, when locating the disk table in windows disk manager, pressing right and left arrows nvda does not read any information.
As I notice when walking with the arrows the discs are selected.
To reproduce:
Press the right button on the item on this computer in windows 10.
Enter the manage option.
Walk with the arrow keys down to Disk Manager.
Now press tabe until the nvda speak "graphical table view".
Now to walk through the table I pressed a down arrow and a right arrow, note that nvda does not say anything, and on my machine when I pressed the right button I was with an external disk selected.

The expected one would be when walking with the arrows the nvda inform the selected discs.

**Comment by ehollig** [comment]
This is a duplacate of #1486. However, an app module was created to work in Windows 7 and it would be interesting to know if it works in Windows 10. Please try it and report your findings in #1486. The app module can be found in https://github.com/nvaccess/nvda/issues/1486#issuecomment-155289857 or you can click here for the [mmc.py.txt](https://github.com/nvaccess/nvda/files/1204321/mmc.py.txt) attachment.
Closing, as discussion will continue in #1486.
